lib/led-matrix.cc

```diff
@@ -183,16 +184,45 @@ uint8_t RGBMatrix::brightness() {
   return brightness_;
 }
 
+void RGBMatrix::SetTransformer(CanvasTransformer *transformer) {
+  transformer_ = transformer;
```

Comment:
I think I'd just keep a dummy transformer around that just returns the canvas itself when someone sets the transformer to NULL.

Otherwise you have to always check for transformer_ != NULL everywhere you want to use it, which is error prone and harder to read.